packages/11ty/_includes/components/accordion/global-controls.js

```diff
@@ -0,0 +1,20 @@
+const { html } = require('~lib/common-tags')
+
+/**
+ * @param {Object} eleventyConfig
+ * @returns Global Accordion Controls Shortcode
+ */
+module.exports = function(eleventyConfig) {
+  const { collapseText, expandText } = eleventyConfig.globalData.config.accordion.globalControls
+
+  /**
+   * @return {String} html markup for global accordion controls
+   */
+  return function() {
+    return html`
+      <div class="global-accordion-controls" data-outputs-exclude="pdf,epub">
```

Comment:
At the page level this could use an `id` attribute, for example
```suggestion
      <div id="accordion-controls" data-outputs-exclude="pdf,epub">
```